fastlane/jetpack_metadata/en-US/release_notes.txt

```diff
@@ -1,18 +1,8 @@
-Only gave the app limited access to your media library? No big deal, but we added a prompt to the media picker in case you want to change your settings and allow more access. The prompt even comes in Dynamic Type for easy reading.
-
-We also made VoiceOver and Dynamic Type more accurate on Activity Log and Schedule Post calendars. We’re happy to report that this improves the user experience for folks who need better visual accessibility.
-
-We made some useful updates to the Block Editor.
-
-- We solved an issue where buttons weren’t justifying left, right, or center in the Buttons block.
-- We added translation for color settings in Cover blocks.
-- Selecting an image will no longer show you a video thumbnail, and vice versa.
-- We hid the help button in the bottom right of the unsupported block editor to avoid confusion.
-- When you’re using different text and background colors in text blocks, you’ll get a notification if the color combo might make it hard for some people to read.
-- We fixed a handful of edge case problems when changing the color of non-selected text.
-
-We also solved a few smaller issues and crashes.
-
-- Weekly roundup notifications are now showing up, making it easier to stay on top of the latest news.
-- When you connect your Facebook account to Publicize, you’ll now be logged in successfully.
-- Images with a copyright symbol in the file name will now upload or appear in the Share Sheet properly.
+* [*] Block editor: Replacing the media for an image set as featured prompts to update the featured image [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930]
+* [***] Block editor: Font size and line-height support for text-based blocks used in block-based themes [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4519]
+* [**] Some of the screens of the app has a new, fresh and more modern visual, including the initial one: My Site. [#17812]
+* [**] Notifications: added a button to mark all notifications in the selected filter as read. [#17840]
+* [**] People: you can now manage Email Followers on the People section! [#17854]
+* [*] Stats: fix navigation between Stats tab. [#17856]
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17888]
+* [*] Stats: Users can now add a new Insights card from the navigation bar. [#17867]
```

Comment:
All the other `release_notes.txt` have been deleted because the automation couldn't find them on GlotPress.

The en-US one is not downloaded (because it will always 404) but `cp`-ed from `WordPress/Jetpack/Resources/release_notes.txt`.